webknossos/webknossos/dataset/dataset.py

```diff
@@ -663,38 +663,38 @@ def from_images(
                 filepaths_per_layer = {
                     f"{layer_name}_{k}": v for k, v in filepaths_per_layer.items()
                 }
-
-        for layer_name, filepaths in filepaths_per_layer.items():
-            filepaths.sort(key=z_slices_sort_key)
-            category: LayerCategoryType
-            if layer_category is None:
-                category = (
-                    "segmentation"
-                    if guess_if_segmentation_path(filepaths[0])
-                    else "color"
+        with get_executor_for_args(None, executor) as executor:
```

Comment:
Can the executor be reused for multiple layers?

Reply:
I added this line, because I ran into an issue like “Cannot schedule new futures after shutdown”. Reusing the executor seems to fix it and enhances the error log.